docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -90,82 +120,74 @@ Consider the following HTML snippet:
       'input.new-todo',
       'placeholder'
   ); <1>
   expect(placeholderValue).toBe('What needs to be done?'); <2>
-})
+});
 ----
-<1> Query the page for the `input.new-todo` selector and get the `placeholder`
-attribute value. See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getAttribute` reference] for more information.
+<1> Find the `input` element with class `new-todo` and get the value of the `placeholder` attribute. See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getAttribute` reference] for more information.
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
 [discrete]
 [[synthetics-sample-test]]
-=== Sample Synthetic test
+=== Sample synthetic test
 
 A complete example of a basic synthetic test looks like this:
 
 [source,js]
 ----
 import { journey, step, expect } from '@elastic/synthetics';
 
-journey("Ensure placeholder is correct", ({page}) => {
-    step("go to todos demo page", async () => {
-      await page.goto('https://elastic.github.io/synthetics-demo/');
-    })
-    step("assert placeholder text", async () => {
-      const placeholderValue = await page.getAttribute(
-        'input.new-todo',
-        'placeholder'
-      );
-      expect(placeholderValue).toBe('What needs to be done?');
-    })
+journey('Ensure placeholder is correct', ({ page }) => {
+  step('Load the demo page', async () => {
+    await page.goto('https://elastic.github.io/synthetics-demo/');
+  });
+  step('Assert placeholder text', async () => {
+    const placeholderValue = await page.getAttribute(
+      'input.new-todo',
+      'placeholder'
+    );
+    expect(placeholderValue).toBe('What needs to be done?');
+  });
 });
 ----
 
 [discrete]
 [[synthetic-run-tests]]
-== Running synthetic tests
-
-As explained in the <<synthetics-quickstart,quickstart>>, there are two ways to run synthetic tests:
-
-* Run as an inline journey -- embed into `heartbeat.yml`
-* Run a test suite -- import your tests into Heartbeat
+== Run synthetic tests
 
-Which option is right for you? That depends.
-If you want to invoke a single journey with few steps, an inline journey might be the easiest workflow --
-you only need to write your tests into your `heartbeat.yml` file.
+There are two ways to run synthetic tests:
 
-Note that you can only paste the steps into an inline script, so these cannot be dependent on any packages that you may want to `import`. If you do have any dependencies, these would need to be defined with an `import` outside of the `journey` object and run via a test suite.
-If you have a complex test suite, or your tests need to live with your application code,
-you're likely better off setting up a synthetic test suite.
+* If you want to invoke a single journey with few steps, <<synthetics-inline-journey,run an inline journey>>.
```

Comment:
I'd probably say that:

> If you'd like to create a single journey and manage it in isolation from monitors containing other journeys, you should probably create an `inline` monitor.

I'd recommend using that phrase because it helps us make it clear to users that whenever they want to manage multiple journeys from a single monitor they should then use ZIP URLs.

In other words, it doesn't really have something to do with the number of steps in a journey, but with the number of journeys per monitor/in a project.